modules/simple_forms_api/app/services/simple_forms_api/form_remediation/form526_submission_remediation_data.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
```

Comment:
This is named `form526_submission_remediation_data.rb`, should it be `form0781_submission_remediation_data.rb`?